common/types/null.go

```diff
@@ -36,6 +39,10 @@ func (n Null) ConvertToNative(refType reflect.Type) (interface{}, error) {
 			Kind: &structpb.Value_NullValue{
 				NullValue: structpb.NullValue_NULL_VALUE}}, nil
 	}
+	if refType == reflect.TypeOf(&any.Any{}) {
+		pb, _ := n.ConvertToNative(jsonValueType)
```

Comment:
Just to be safe, you should probably check that there isn't an error converting the value to JSON. If there is, I would return it instead.

Reply:
done.